api/src/opentrons/protocol_api/labware.py

```diff
@@ -1093,14 +1103,17 @@ def filter_tipracks_to_start(
 
 # TODO(mc, 2022-11-09): implementation detail, move to core
 def next_available_tip(
-    starting_tip: Optional[Well], tip_racks: List[Labware], channels: int
+    starting_tip: Optional[Well],
+    tip_racks: List[Labware],
+    channels: int,
+    nozzle_map: Optional[NozzleMap] = None,
```

Comment:
Ditto other comment about making this new argument keyword-only. (I'm conservatively assuming this function is just as public, but I haven't checked and it might not be.)

```suggestion
    starting_tip: Optional[Well],
    tip_racks: List[Labware],
    channels: int,
    *,
    nozzle_map: Optional[NozzleMap] = None,
```